tests/io/test_eleanorlite.py

```diff
@@ -0,0 +1,37 @@
+import pytest
+
+from astropy.io import fits
+import numpy as np
+from numpy.testing import assert_array_equal
+
+from lightkurve import search_lightcurve
+from lightkurve.io.eleanorlite import read_eleanorlite_lightcurve
+from lightkurve.io.detect import detect_filetype
+
+@pytest.mark.remote_data
+def test_eleanor_lite():
+    """Can we read in GSFC-ELEANOR-LITE light curves?"""
+    url = "https://mast.stsci.edu/api/v0.1/Download/file?uri=mast:HLSP/gsfc-eleanor-lite/s0001/0000/0003/3673/2616/hlsp_gsfc-eleanor-lite_tess_ffi_s0001-0000000336732616_tess_v1.0_lc.fits" 
```

Comment:
- please remove trailing whitespace
- Style: For long URL, we  generally break it down so that each line is < 127 characters (usually < 100 characters if possible). See `test_lightcurve.py` for an example.
  - In general, we also try to limit each line to be less than 127 characters (preferrably less than 100 characters), for readability / ease of review, e.g., avoid the need of horizontal scroll when one reviews a PR. But this is not strictly enforced.